lib/Echidna/ABI.hs

```diff
@@ -241,20 +233,37 @@ shrinkAbiCall = traverse $ traverse shrinkAbiValue
 
 -- | Given an 'AbiValue', generate a random \"similar\" value of the same 'AbiType'.
 mutateAbiValue :: MonadRandom m => AbiValue -> m AbiValue
-mutateAbiValue (AbiUInt n x)         = AbiUInt n         <$> mutateNum x
-mutateAbiValue (AbiInt n x)          = AbiInt n          <$> mutateNum x
-mutateAbiValue (AbiAddress _)        = genAbiValue AbiAddressType
+mutateAbiValue (AbiUInt n x)         = getRandomR (0, 9 :: Int) >>= -- 10% of chance of mutation
+                                          \case  
+                                            0 -> (AbiUInt n <$> mutateNum x)
+                                            _ -> return $ AbiUInt n x
+mutateAbiValue (AbiInt n x)          = getRandomR (0, 9 :: Int) >>= -- 10% of chance of mutation
+                                          \case  
+                                            0 -> (AbiInt n <$> mutateNum x)
+                                            _ -> return $ AbiInt n x 
```

Comment:
can we make these 10 percents tunable config parameters. if you want to go the `Rational` route the JSON for that is `{ "numerator" : 1, "denominator" : 10 }`. otherwise choose something like `Double`.

Reply:
Increasing these values will make the tests to fail, because magic numbers are going to be (likely) corrupted. I think the user should be allowed to modify these.